awscli/customizations/emr/emrutils.py

```diff
@@ -153,20 +154,13 @@ def check_empty_string_list(name, value):
         raise exceptions.EmptyListError(param=name)
 
 
-def call(session, operation_object, parameters, region_name=None,
+def call(session, operation, parameters, region_name=None,
          endpoint_url=None, verify=None):
-        # We could get an error from get_endpoint() about not having
-        # a region configured.  Before this happens we want to check
-        # for credentials so we can give a good error message.
-        if session.get_credentials() is None:
-            raise NoCredentialsError()
-
-        endpoint = operation_object.service.get_endpoint(
-            region_name=region_name, endpoint_url=endpoint_url,
+        client = session.create_client(
+            'emr', region_name=region_name, endpoint_url=endpoint_url,
             verify=verify)
-        LOG.debug('Calling ' + str(operation_object) + ' with endpoint: ' +
```

Comment:
I guess that I can keep the log message since I have ``client.meta.endpoint_url``. I feel knowing the endpoint_url is not really important since it is logged in botocore.